server/src/main/java/io/kroki/server/service/Structurizr.java

```diff
@@ -48,6 +50,7 @@ public class Structurizr implements DiagramService {
 
   public Structurizr(Vertx vertx, JsonObject config) {
     this.vertx = vertx;
+    this.safeMode = SafeMode.get(config.getString("KROKI_STRUCTURIZR_RESTRICTED_PARSER", "secure"), SafeMode.SECURE);
```

Comment:
To be honest, I hadn't thought about reusing the concept of safe mode. I had envisioned a configuration that would restrict the parser with a boolean value. If we want to reuse the concept of safe mode, I suppose we should do the following:

- Rename the environment variable to `KROKI_STRUCTURIZR_SAFE_MODE`
- Use the value of `KROKI_SAFE_MODE` if `KROKI_STRUCTURIZR_SAFE_MODE` is not defined
- Explain this functionality in the documentation: https://github.com/yuzutech/kroki/blob/358075666a392e1f58885f6340344b761b5a037c/docs/modules/setup/pages/configuration.adoc?plain=1#L47


What do you think?